src/components/__tests__/Editable.js

```diff
@@ -0,0 +1,99 @@
+import { store } from '../../store'
+import { createTestApp } from '../../setupTests'
+import { RANKED_ROOT, ROOT_TOKEN } from '../../constants'
+import { equalArrays, pathToContext } from '../../util'
+import { importText } from '../../action-creators'
+import Editable from '../Editable'
+import ContentEditable from 'react-contenteditable'
+import { noop } from 'lodash'
+import { exportContext } from '../../selectors'
+
+/** A filterWhere predicate that returns true for Thought or Subthought nodes that match the given context. */
+const whereContext = context => node => equalArrays(pathToContext(node.props().thoughtsRanked), context)
+
+beforeAll(async () => {
+  await createTestApp()
+})
+
+afterEach(async () => {
+  store.dispatch({ type: 'clear' })
+})
+
+it('indent on adding space at the beginning of the thought', async () => {
+
+  await store.dispatch(importText(RANKED_ROOT, `
+    - a
+      - b
+        - c
+        - d`))
+
+  await store.dispatch({
+    type: 'setCursor',
+    thoughtsRanked: [{ value: 'a', rank: 0 }, { value: 'b', rank: 1 }, { value: 'd', rank: 3 }],
+    flag: true })
```

Comment:
Was `flag` used for debugging?

Reply:
Yes. I forgot to remove it. I will do that.